modules/audio_device/android/java/src/org/webrtc/voiceengine/WebRtcAudioTrack.java

```diff
@@ -154,7 +157,7 @@ public void run() {
           byteBuffer.put(emptyBytes);
           byteBuffer.position(0);
         }
-        int bytesWritten = writeBytes(audioTrack, byteBuffer, sizeInBytes);
+        int bytesWritten = audioTrack.write(byteBuffer, sizeInBytes, AudioTrack.WRITE_BLOCKING);
```

Comment:
Might we want to keep the writeBytes() function around so that API < 21 are still supported?

Reply:
Fixed